### PR TITLE
Add Strings Example

### PR DESCRIPTION
**Pull request by rcatalan98**

### CODE REVIEW

File: content/examples/strings.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: "strings"
+date: 2023-06-21
+draft: false
+---
+In Cairo a string is a collection of characters stored in a Felt252. Strings can have a maximum length of 31 characters.
```
Comment (MegaRedHand) [inline content/examples/strings.md:6]:
```suggestion
In Cairo, a string is a collection of characters stored in a Felt252. Strings can have a maximum length of 31 characters.
```

File: content/examples/strings.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+```rust {.codebox}
+use debug::PrintTrait;
+fn main() {
+    let x = 'Cairo is awesome';
+    x.print();
+    let c = 'A';
+    c.print();
+}
+```
+Run ```cairo-run file_name```
```
[the remark this answers is not in the document]
Comment (rcatalan98) [reply content/examples/strings.md:19]: The idea was to have the command of how to run each program (they should copy the code) so it's easy and fast to try each example. If it's confusing we can remove it.

[the remark this answers is not in the document]
Comment (MegaRedHand) [reply content/examples/strings.md]: Oh, ok. I think we should leave it, but maybe adding a "To run this example, copy the code in `file.cairo` and run it with:"

Also, the templates we are using allow us to include this in every example, so in the future we could normalize this.

[the remark this answers is not in the document]
Comment (rcatalan98) [reply content/examples/strings.md]: Great idea. I'll take a look at that.

**Review by MegaRedHand — APPROVED** [review]
👍

**Pull request merged**